fsimpl/localfs/localfs.go

```diff
@@ -160,7 +156,9 @@ func (l *Local) GetAttr(req p9.AttrMask) (p9.QID, p9.AttrMask, p9.Attr, error) {
 // Close implements p9.File.Close.
 func (l *Local) Close() error {
 	if l.file != nil {
-		return l.file.Close()
+		err := l.file.Close()
+		l.file = nil
+		return err
```

Comment:
Something paranoid came to mind, so I'll mention it but it's not a suggestion.

If `file.Close` panics, `Local` will retain the reference to `file`.
Something like this would guarantee `file` gets dropped/invalidated (and indirectly assure `file.Close` only gets called once).
```go
if file := l.file; file != nil {
	l.file = nil
	return file.Close()
}
```